docs/excel/add-properties-to-basic-cell-values.md

```diff
@@ -0,0 +1,398 @@
+
+```javascript
+async function addPropertyToNumber() {
+  await Excel.run(async (context) => {
+    let sheet = context.workbook.worksheets.getActiveWorksheet();
+    let range = sheet.getRange("A1");
+    range.load("valuesAsJson");
+    await context.sync();
+    let cellValue = range.valuesAsJson[0][0] as any;
+
+    // Only apply this property to a double.
+    if (cellValue.basicType === "Double") {
+      cellValue.properties = {
+        Precision: {
+          type: Excel.CellValueType.double,
+          basicValue: 4
+        }
+      };
+      range.valuesAsJson = [[cellValue]];
+      await context.sync();
+    }
+  });
+}
+```
+
+## Differences from entity values
+
+Adding properties to **string**, **Boolean**, and **double** basic types is similar to adding properties to entity values. However there are differences.
+
+- Basic types have a non-error fallback so that calculations can operate on them. For example, consider the formula **=SUM(A1:A3)** where **A1** is **1** (a double with properties), **A2** is **2**, and **A3** is **3**. The sum will return the correct result of **6**. The formula would not work if **A1** was an entity value.
+- When the value of a basic type is used in a calculation, the properties are discarded. In the previous example of **=SUM(A1:A3)** where A1 is a double with properties, the result of **6** will not have any properties.
+- If no icon is specified for a basic type, the cell does not show any icon. If an entity value does not specify an icon, it will still show a default icon in the cell value.
+
+## Formatted number values
+
+You can apply number formatting to values of type **CellValueType.double**. Use the **numberFormat** property in the JSON schema to specify a number format. The following code sample shows the complete schema of a number value formatted as currency. The formatted number value in the code sample displays as **$24.00** in the Excel UI.
+
+```javascript
+// This is an example of the complete JSON of a formatted number value with a property.
+// In this case, the number is formatted as currency.
+async function createCurrencyValue() {
+  await Excel.run(async (context) => {
+    const sheet = context.workbook.worksheets.getActiveWorksheet();
+    const range = sheet.getRange("A1");
+    range.valuesAsJson = [
+      [
+        {
+          type: Excel.CellValueType.double,
+          basicType: Excel.RangeValueType.double,
+          basicValue: 24,
+          numberFormat: "$0.00",
+          properties: {
+            Name: {
+              type: Excel.CellValueType.string,
+              basicValue: "dollar"
+            }
+          }
+        }
+      ]
+    ];
+    await context.sync();
+  });
+}
+```
+
+The number formatting is considered the default format. If the user, or other code, applies formatting to a cell containing a formatted number, the applied format will override the number’s format.
+
+## Card layout
```

Comment:
Broad question: 
It looks like the next three sections are very similar to sections in the existing [Entity value cards](https://learn.microsoft.com/en-us/office/dev/add-ins/excel/excel-data-types-entity-card) article. Is the main difference that these new sections describe how to add properties to CellValueTypes other than `entity`? If so, is separating the articles into entity/not-entity like this, the most logical? Should we have an article for each type? Such as one article each for entity, double, string, etc.